src/main/java/org/osc/sdk/controller/element/InspectionPortElement.java

```diff
@@ -23,7 +23,7 @@
  * If the ingress and egress ports are the same, then the same port will need to be returned for either of the methods.
  */
 @ConsumerType
-public interface InspectionPortElement {
+public interface InspectionPortElement extends Element {
```

Comment:
suggestion: can we update the comment for this interface definition saying it now has Element has it member?.